packages/blaise/src/index.ts

```diff
@@ -0,0 +1,89 @@
+import { AvroKafkaMessage } from '@ovotech/avro-kafkajs/dist/types';
+import avroToJson from '@ovotech/avro-mock-generator';
+import {
+  CastleEachBatchPayload,
+  CastleEachMessagePayload,
+} from '@ovotech/castle';
+import merge = require('lodash.merge');
+import { DeepPartial } from 'ts-essentials';
+import { BlaiseDefaults, IBlaise } from './types';
+
+// tslint:disable-next-line
```

Comment:
Can we use eslint, as that's what the other packages here are using and it is, you know ["the future"](https://medium.com/palantir/tslint-in-2019-1a144c2317a9).  Also might be worth disabling the specific error not linting altogether